ios/RCTMapboxGL/RCTConvert+MapKit.m

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTConvert+MapKit.h"
+#import "RCTConvert+CoreLocation.h"
```

Comment:
This file hasn't been copied into the codebase - you probably want `import <React/RCTConvert+CoreLocation.h>` instead.